src/components/cursor.js

```diff
@@ -33,60 +38,118 @@ module.exports.Component = registerComponent('cursor', {
   dependencies: ['raycaster'],
 
   schema: {
+    downEvents: {default: []},
     fuse: {default: utils.device.isMobile()},
-    fuseTimeout: {default: 1500, min: 0}
+    fuseTimeout: {default: 1500, min: 0},
+    laserAlwaysShow: {default: true},
```

Comment:
I don't see where `laserAlwaysShow` is used.

just thinking out loud, but what if – borrowing from CSS' `border` property – there were these properties?

- laserColor
- laserWidth
- laserStyle

if `laserWidth <= 0`, then there's no laser. `laserStyle` could indicate whether it's visible or not.